### PR TITLE
Improved text and links in the Matomo Help Centre

### PR DESCRIPTION
**Pull request by snake14**
### Description:

We are updating some of the text and links in the Matomo Help Centre to try and make it a little more user friendly and informative.
Jira Issue: PG-1320

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/Feedback/lang/en.json [hunk 1]
```diff
@@ -38,7 +39,7 @@
         "TopLinkTooltip": "Tell us what you think, or request Professional Support.",
         "ViewAnswersToFAQ": "View answers to %1$sFrequently Asked Questions%2$s",
         "FAQs": "FAQs",
-        "ViewUserGuides": "Learn how to configure Matomo and how to effectively analyze your data with our %1$suser guides%2$s",
+        "ViewUserGuides": "FAQs, comprehensive guides, how-to’s and troubleshooting documents.",
```
Comment (justinvelluppillai) [inline plugins/Feedback/lang/en.json]: We can't replace a translation string with placeholders with one without. We need to create a new key for the new translation string without the placeholder, and remove the old one if no longer needed. This way weblate will handle the translations correctly for other languages.

Comment (snake14) [reply plugins/Feedback/lang/en.json]: Thank you. I just reverted a couple translations and created new ones instead.

File: plugins/Feedback/templates/index.twig [hunk 2]
```diff
@@ -21,16 +21,17 @@
...
-            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/faq/?mtm_campaign=App_Help&mtm_source=Matomo_App&mtm_keyword=FAQs">{{ 'Feedback_FAQs'|translate }}</a> - {{ 'Feedback_ViewAnswersToFAQ'|translate("","") }}.</p>
-            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://forum.matomo.org/">{{ 'Feedback_Forums'|translate }}</a> - {{ 'Feedback_VisitTheForums'|translate("","") }}.</p>
-            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="{{ linkTo({'module':"API",'action':"glossary"}) }}&mtm_campaign=App_Help&mtm_source=Matomo_App&mtm_keyword=glossary">{{ 'API_Glossary'|translate }}</a> - {{ 'API_LearnAboutCommonlyUsedTerms2'|translate }}</p>
+            <h4>{{ 'Feedback_MatomoHelpCentre'|translate }}</h4>
+            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/guides/">{{ 'Feedback_KnowledgeBase'|translate }}</a> - {{ 'Feedback_KnowledgeBaseDescription'|translate }}</p>
```
Comment (peterhashair) [inline plugins/Feedback/templates/index.twig]:
```suggestion
            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/guides/?mtm_campaign=App_Help&mtm_source=Matomo_App&mtm_keyword=guide">{{ 'Feedback_KnowledgeBase'|translate }}</a> - {{ 'Feedback_KnowledgeBaseDescription'|translate }}</p>
```

File: plugins/Feedback/templates/index.twig [hunk 3]
```diff
@@ -21,16 +21,17 @@
...
+            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/web-analytics-training/">{{ 'Feedback_MatomoVideoTraining'|translate }}</a> - {{ 'Feedback_MatomoVideoTrainingDescription'|translate }}</p>
```
Comment (peterhashair) [inline plugins/Feedback/templates/index.twig]:
```suggestion
            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/web-analytics-training/?mtm_campaign=App_Help&mtm_source=Matomo_App&mtm_keyword=docs">{{ 'Feedback_MatomoVideoTraining'|translate }}</a> - {{ 'Feedback_MatomoVideoTrainingDescription'|translate }}</p>
```

File: plugins/Feedback/templates/index.twig [hunk 4]
```diff
@@ -21,16 +21,17 @@
...
+            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/tag-manager-training/">{{ 'Feedback_MtmVideoTraining'|translate }}</a> - {{ 'Feedback_MtmVideoTrainingDescription'|translate }}</p>
```
Comment (peterhashair) [inline plugins/Feedback/templates/index.twig]:
```suggestion
            <p> &bull; <a target="_blank" rel="noreferrer noopener" href="https://matomo.org/tag-manager-training/?mtm_campaign=App_Help&mtm_source=Matomo_App&mtm_keyword=docs">{{ 'Feedback_MtmVideoTraining'|translate }}</a> - {{ 'Feedback_MtmVideoTrainingDescription'|translate }}</p>
```

**Review by peterhashair — COMMENTED** [review]
looks great, but maybe just add some campaign param, to help us tracking the article.

**Comment by snake14** [comment]
Thank you @peterhashair  . I have added the recommended changes.

**Comment by sgiehl** [comment]
@snake14 If you remove the usage of a translation, you should check if they are in use anywhere else. If they are unused afterwards you can also remove them from `en.json`.

**Comment by snake14** [comment]
> @snake14 If you remove the usage of a translation, you should check if they are in use anywhere else. If they are unused afterwards you can also remove them from `en.json`.

Thank you @sgiehl . I went ahead and removed the translation key that isn't being used anymore.

**Review by sgiehl — APPROVED** [review]

**Comment by sgiehl** [comment]
code looks fine so far. I've merged in the latest changes from 4.x-dev.
If tests are passing, this should be good to merge.

**Comment by sgiehl** [comment]
test failures are unrelated.

**Comment by snake14** [comment]
@sgiehl Thank you for your help :+1:

**Pull request merged**